### ISSUE TITLE
Not working on Oculus Go

### ISSUE THREAD

**Issue by Everlight-Helios** [Unity-Technologies/PostProcessing#632]
##### What happened?
Tried to use a depth of field effect and a bloom effect, when built out to Oculus Go, the left eye shows a completely grey screen, while the right shows the scene without effects and an extremely hard lag and several screen tears. This while the Post-Process Layer is on the center eye anchor camera.

##### Post-processing stack version (v1, v2)?
v2, I downloaded the stack today.

##### Unity version, operating system, target platform (standalone windows, mac, iOS, PS4...)?
Unity 2017.4.1, Windows 10, Oculus Go (Android)

**Comment by franadoriv** [comment]
It doesnt work neither on 2018.2.2f1. 
I have the same behavior. Full yellow on the left eye, slow scene without any effect on the right.
The problem is with Single-Pass Stereo rendering, otherwise works fine but without this option everything become slower. There is no workaround for this? I need this urgent

**Comment by andybak** [comment]
There's a branch where it's been worked on: https://github.com/Unity-Technologies/PostProcessing/commits/xr-spi

However @robinb-u3d hasn't responded to a comment from me asking for some more info. Maybe we need to prod someone on Twitter...

**Comment by Dmart331** [comment]
Just started putting post processing effects on my project with the go and I'm getting the same issue.. I'm guessing with this being open they still haven't addressed it?

**Comment by AlexandreFiset** [comment]
Yup. Still not working. Several months owning an Oculus Go and never got post processing to work on it. This is a shame. Tested again today with the latest master and package manager versions.

**Comment by 13Flo** [comment]
Please add your vote here:
https://issuetracker.unity3d.com/issues/oculus-go-postprocessing-effects-are-not-applied-to-built-applications

**Comment by Fragilem17** [comment]
and here (same issue most likely): https://issuetracker.unity3d.com/issues/vr-android-depth-map-right-eye-issue-with-vr-single-pass-sampling-cameradepthtexture

**Comment by viknesh2020** [comment]
The issue @Everlight-Helios mentioned still exists in 2019.1.10 with PPS v2.1.7 installed from Package Manager.

**Comment by mchangxe** [comment]
Can confirm this also still exists in 2019.1.10 with newest pps. However, instead of one camera as grey, I get a rectangle string going down the center of the screen rending at a different angle of the scene. Apart from that, both eyes work well.
